client/src/layout/BurgerMenu/BurgerMenu.js

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+import Box from '@mui/material/Box';
+import IconButton from '@mui/material/IconButton';
+import Menu from '@mui/material/Menu';
+import MenuIcon from '@mui/icons-material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import { NavLink } from 'react-router-dom';
+import styles from './BurgerMenu.module.scss';
+
+const pages = [
+  {
+    name: 'Home',
+    link: '/',
+    innerContent: 'Home',
+  },
+  {
+    name: 'Catalogue',
+    link: '/catalogue',
+    innerContent: 'Catalogue',
+  },
+  {
+    name: 'Cart',
+    link: '/cart',
+    innerContent: 'Cart',
+  },
+];
+
+const BurgerMenu = () => {
+  const [anchorElNav, setAnchorElNav] = useState(null);
+
+  const handleOpenNavMenu = (event) => {
+    setAnchorElNav(event.currentTarget);
+  };
+
+  const handleCloseNavMenu = () => {
+    setAnchorElNav(null);
+  };
+
+  const boxSX = {
+    flexGrow: 1,
+    display: { sx: 'flex', laptop: 'none' },
+    justifyContent: 'flex-end',
+  };
+
+  const menuIconSX = {
+    '&:hover': {
+      color: 'yellow',
```

Comment:
`color: '#FFC700',`

please fix color

Reply:
> `color: '#FFC700',`
> 
> please fix color

I think it is better to use 'color: "secondary.main"'

Reply:
i think we don’t  need hover effect on burger menu, because it’s icon for tablets and mobiles

Reply:
@RomanRymarchuk1  i agree. no need for hover on touch screen devices.

Reply:
done